### PR TITLE
pushLocal.sh script

### LINKED ISSUE #1149: Way to copy portal source into PFE for debugging (for devs)
**Codewind version:** 0.7.0
**OS:**

**Che version:**
**IDE extension version:**
**IDE version:**
**Kubernetes cluster:**

**Description of the enhancement:**

Recently, the `start.sh` script used by developers was changed to use `cwctl` to start up codewind https://github.com/eclipse/codewind/pull/1085

However, now it seems that it's no longer possible to add the portal source folder as a mount into PFE; that ability was very useful for developers as changes made to the source would trigger `nodemon` to restart the process and pick up the changes, without needing to rebuild and restart.

**Proposed solution:** 

A way to pass in the location of portal source folder (possibly only when `--debug` is used) and mount it to PFE when starting it

### PR DESCRIPTION
**Pull request by makandre**
Signed-off-by: Andrew Mak <makandre@ca.ibm.com>

Resolves #1149 
This PR supersedes #1275 

Adding a `pushLocal` script as suggested by @rajivnathan 

@tobespc @rajivnathan pls review, thanks.

### CODE REVIEW

File: src/pfe/portal/package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "dev": "nodemon --inspect=0.0.0.0:9777 server.js",
     "coverage": "nyc node server.js",
     "local": "node server.js --local",
-    "eslint": "eslint ."
+    "eslint": "eslint .",
+    "pushLocal": "./pushLocal.sh"
```
Comment (rajivnathan) [inline src/pfe/portal/package.json]: You may want to run eslint as part of the pushLocal

**Comment by johnmcollier** [comment]
Wondering if we should move the script one level up into the PFE folder rather than storing it directly as part of the Portal container's code?

**Comment by makandre** [comment]
added to `.dockerignore`

File: src/pfe/portal/pushLocal.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+docker cp config/. codewind-pfe:/portal/config
+docker cp middleware/. codewind-pfe:/portal/middleware
+docker cp modules/. codewind-pfe:/portal/modules
+docker cp routes/. codewind-pfe:/portal/routes
+docker cp server.js codewind-pfe:/portal/server.js
```
Comment (hhellyer) [inline src/pfe/portal/pushLocal.sh]: Do you need to add the `openapi.yml` file here? It's copied in when we do a build and is actually used by the `validateReq` middleware function?

**Comment by makandre** [comment]
@hhellyer Ok added the docs folder and will mention this script in the next playback

**Pull request merged**